clickable.yaml

```diff
@@ -1,47 +1,40 @@
 clickable_minimum_required: '7'
 builder: go
+prebuild: mkdir -p ${BUILD_DIR}/axolotl-web && cp ${ROOT}/axolotl-web/dist/* ${BUILD_DIR}/axolotl-web -R
 postbuild: ${ROOT}/scripts/postbuild.sh ${INSTALL_DIR};
 install_bin:
   - ${CRAYFISH_LIB_INSTALL_DIR}/bin/crayfish
+install_root_data:
+  - manifest.json
+  - click/axolotl.png
+  - click/textsecure-helper
+  - click/textsecure-push-helper.json
+  - click/textsecure-push.apparmor
+  - click/textsecure.apparmor
+  - click/textsecure.content-hub
+  - click/textsecure.desktop
+  - click/textsecure.png
+  - click/textsecure.url-dispatcher
+  - ${BUILD_DIR}/axolotl-web
```

Comment:
Where is $BUILD_DIR coming from? And from what I understand, axolotl-web is always at the same location - do we need a variable here or could we just hard-code the location?

Reply:
`BUILD_DIR` is [provided by Clickable](https://clickable-ut.dev/en/dev/project-config.html#placeholders-environment-variables). See the `prebuild` command for how `axolotl-web` gets there. The goal is to rename some content. The alternative would be to add the axolotl-web from the source dir here and rename it in `postbuild`.

Reply:
` $BUILD_DIR ` is a specific value in the build process of clickable: see in the documentation [here](https://clickable-ut.dev/en/dev/project-config.html?highlight=BUILD_DIR#build-dir) and [here](https://clickable-ut.dev/en/dev/project-config.html?highlight=BUILD_DIR#placeholders-environment-variables)
My idea was to have a unique build dir where all the necessary parts are in before the clickpackage is produced.

`prebuild: mkdir -p ${BUILD_DIR}/axolotl-web && cp ${ROOT}/axolotl-web/dist/* ${BUILD_DIR}/axolotl-web -R` 

Another reason is, a running axolotl instance is looking at the path `./axolotl-web` for the html files and when we copy from src directory it's in `./dist` because the  `install_root_data:` copies the named files into the root folder. For the renaming I added the prebuild script.